modules/server/src/main/scala/seqexec/server/tcs/TcsSettleTimeCalculator.scala

```diff
@@ -41,7 +41,7 @@ object TcsSettleTimeCalculator {
   val settleTimeCalculators: Map[Subsystem, SettleTimeCalculator] = Map(
     Subsystem.Mount -> constantSettleTime(1.seconds),
     Subsystem.PWFS1 -> constantSettleTime(1.seconds),
-    Subsystem.PWFS2 -> constantSettleTime(1.seconds)
+    Subsystem.PWFS2 -> constantSettleTime(2.seconds)
```

Comment:
- [ ] Will this apply to GN and GS? Wasn't the smearing mostly in the south?

Reply:
It was detected at GS when using F2 + PWFS2. Yes, the change will affect both sites.